src/main/java/io/rtr/conduit/amqp/impl/AMQPAsyncConsumerBuilder.java

```diff
@@ -36,7 +36,10 @@ protected AMQPAsyncListenProperties buildListenProperties() {
                 shouldPurgeOnConnect(),
                 isDynamicQueueCreation(),
                 getPoisonPrefix(),
-                getDynamicQueueRoutingKey());
+                getDynamicQueueRoutingKey(),
+                isEnsureBasicConfig(),
```

Comment:
basic config as a name to me implies something generic but this change is aimed at ensuring that your queue is created and bound.  could you rename?